src/v8/update-mui-dependencies.ts

```diff
@@ -13,11 +13,11 @@ export default async function updateMuiDependencies() {
 
     const packageJson = new PackageJson(adminPackageJsonPath);
 
-    packageJson.addDependency("@mui/material", "^6.0.0");
-    packageJson.addDependency("@mui/system", "^6.0.0");
-    packageJson.addDependency("@mui/utils", "^6.0.0");
-    packageJson.addDependency("@mui/icons-material", "^6.0.0");
-    packageJson.addDependency("@mui/lab", "^6.0.0-beta.10");
+    packageJson.updateDependency("@mui/material", "^7.0.0");
+    packageJson.updateDependency("@mui/system", "^7.0.0");
+    packageJson.updateDependency("@mui/utils", "^7.0.0");
+    packageJson.updateDependency("@mui/icons-material", "^7.0.0");
+    packageJson.updateDependency("@mui/lab", "^7.0.0-beta.9");
```

Comment:
Do we need the lab package? I believe all components we're using have been moved to the core package. Maybe we can remove it?

Reply:
In Comet itself: yes, but if the package is explicitly installed in a project, possibly for other components needed by that project it would be wrong to remove the package.